### ISSUE TITLE
macOS 14.0 - Permission Denied after 14->12 trick

### ISSUE THREAD

**Issue by tobybot11** [keybase/client#25952]
macOS Sonoma 14.0

kbfs volumes mounted after doing the symlink 14->12 tricks in 
/Applications/Keybase.app/Contents/Resources/KeybaseInstaller.app/Contents/Resources/kbfuse.bundle/Contents/Extensions

but now I get Permission Denied when trying to access the /Volumes/Keybase (<username>) and /Volumes/Keybase directories.

both mount points are set to 600 

my log id: 565b568f95201c5ba8e84e1c

**Comment by nilicule** [comment]
Duplicate of #25718

**Comment by GwynethLlewelyn** [comment]
Indeed. I wonder if people even bother to search the issues before posting. We've getting lots of those lately.
